lib/pacemaker/pcmk_verify.c

```diff
@@ -25,27 +25,25 @@
 int
 pcmk__parse_cib(pcmk__output_t *out, const char *cib_source, xmlNodePtr *cib_object)
 {
-    int rc = pcmk_rc_ok;
+    // @COMPAT Take an enum for cib_source instead of trying to figure it out?
```

Comment:
Yeah it's not a big deal. As a note, taking an enum is arguably simpler even if we don't change the public API. There are only two callers (`pcmk_verify()` and `crm_verify.c:main()`).
* `pcmk_verify()` takes arbitrary string input that may be a filename or raw XML (can't change that til a break).
* `crm_verify` knows based on options what type of `cib_source` it's passing to `pcmk__parse_cib()`.

It would be a sort of separation of concerns, and technically it would save work for `crm_verify` (though it's trivial).